src/fparser/__init__.py

```diff
@@ -65,11 +65,24 @@
 # First version by: Pearu Peterson <pearu@cens.ioc.ee>
 # First created: Oct 2006
 
+try:
+    from importlib.metadata import version, PackageNotFoundError
+except ModuleNotFoundError:
+    # Use backport package for python <3.8
+    from importlib_metadata import version, PackageNotFoundError
+try:
+    __version__ = version(__name__)
+except PackageNotFoundError:
+    from setuptools_scm import get_version
+
+    __version__ = get_version(root="../..", relative_to=__file__)
+
 import logging
 import codecs
 
-logging.getLogger(__name__).addHandler(logging.NullHandler())
+__all__ = ["__version__"]
```

Comment:
Please add a comment saying what "__version__" is used for.

Reply:
This use is really just to shut up the linter about unused variables. Perhaps it would be better to annotate the lines directly (`# noqa` I think for flake8?)

Reply:
Yes, I think that would be better.